src/index.js

```diff
@@ -46,47 +44,35 @@ module.exports = {
     'id-length': ['error', {
       exceptions: ['_', 'e', 'i']
     }],
+    'id-match': ['error', '^[_a-zA-Z][a-zA-Z0-9]*$|^[A-Z][A-Z0-9]*_[A-Z0-9]+$'],
     indent: ['error', 2, {
       SwitchCase: 1
     }],
-    'key-spacing': 'off',
-    'keyword-spacing': ['off', {
-      overrides: {
-        case: {
-          after: true
-        },
-        return: {
-          after: true
-        },
-        throw: {
-          after: true
-        }
-      }
-    }],
-    'linebreak-style': 'off',
+    'key-spacing': 'error',
+    'keyword-spacing': 'error',
+    'linebreak-style': 'error',
     'lines-around-comment': 'off',
     'max-depth': 'error',
     'max-nested-callbacks': 'off',
     'max-params': ['error', 4],
-    'new-cap': 'off',
+    'mocha/no-exclusive-tests': 'error',
+    'new-cap': 'error',
     'new-parens': 'error',
     'newline-after-var': 'off',
     'no-alert': 'error',
     'no-array-constructor': 'error',
     'no-bitwise': 'error',
     'no-caller': 'error',
     'no-catch-shadow': 'off',
-    'no-class-assign': 'error',
     'no-cond-assign': ['error', 'always'],
     'no-confusing-arrow': 'error',
-    'no-const-assign': 'error',
-    'no-constant-condition': 'error',
     'no-div-regex': 'error',
-    'no-dupe-class-members': 'error',
     'no-duplicate-imports': 'error',
     'no-else-return': 'error',
+    'no-empty': ['error', {
+      allowEmptyCatch: true
```

Comment:
If we're strictly porting the jscs rules with minimal modifications, then it makes sense to keep this, since that's the behaviour of the jscs rule: http://jscs.info/rule/disallowEmptyBlocks.